### DISCUSSION TITLE
API exists?

### DISCUSSION THREAD

**Discussion by TheLink1999** [sumatrapdfreader/sumatrapdf#3120] [Documentation]
I was search, but don't find any API for using SumatraPdf functionality by any programming language. If there exists, please add link of that documentation.

Thanks in advance.

**Comment by GitHubRulesOK** [comment]
SumatraPDF is not an API but uses MuPDF which is also offered as PyMuPDF so for most C developers you need to use [Artifex MuPDF](https://github.com/ArtifexSoftware/mupdf) or for hobbyists there is https://github.com/pymupdf/PyMuPDF for variants see https://github.com/search?l=Java&q=mupdf&type=Repositories
